src/Device.ts

```diff
@@ -100,7 +100,14 @@ class Main {
                         locationCapability: RegulatoryLocationType.IndoorOutdoor,
                         supportsConcurrentConnections: true,
                     }, GeneralCommissioningClusterHandler),
-                    new ClusterServer(OperationalCredentialsCluster, {},
+                    new ClusterServer(OperationalCredentialsCluster, {
+                            nocs: [],
+                            fabrics: [],
+                            supportedFabrics: 254,
+                            commissionedFabrics: 0,
+                            trustedRootCertificates: [],
+                            currentFabricIndex: 0,
+                        },
```

Comment:
No idea what we need to use as defaults here :-( but with compleing the spoecs we need to set it ... 

I need to test how chiptool reacts still, will do tomorrow